README.md

```diff
@@ -14,6 +14,7 @@ SwiftUI, UIKit, and more, and on any Apple platform (iOS, macOS, tvOS, and watch
 * [Examples](#examples)
 * [Basic usage](#basic-usage)
 * [Documentation](#documentation)
+* [Migration Guides](https://pointfreeco.github.io/swift-composable-architecture/main/documentation/composablearchitecture/migrationguides)
```

Comment:
This area of the README is more a table of contents, but I think we'd be down to add a direct link to each migration guide down in the "Documentation" section, _e.g._:

---

- [1.8.0](#todo) ([migration guide](#todo))

---

Wanna give that a shot?

Reply:
Sure. Give me a bit today.